app/views/items/edit.erb

```diff
@@ -1,9 +1,19 @@
-<form action="/items/<%= @item.id %>" method="POST">
-  <p>Edit</p>
-  <input type="hidden" name="_method" value="PUT" />
-  <input type='text' name='item[name]' value="<%= @item.name %>"/><br/>
-  <input type='text' name='item[description]' value="<%= @item.name %>"/><br/>
-  <input type='decimal' name='item[price]' value="<%= @item.name %>"/><br/>
-  <input type='text' name='item[image]' value="<%= @item.name %>"/><br/>
-  <input type='submit'/>
-</form>
+<!DOCTYPE html>
+<html>
```

Comment:
I'm curious why some files set the language to default to english, and others don't.